src/main/com/example009/radek/v3/Calculator.java

```diff
@@ -0,0 +1,66 @@
+package main.com.example009.radek.v3;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+class Calculator {
+
+    private double result;
+    private final FileService fileService = new FileService();
+    private List<Double> list = new ArrayList<>();
+
+    Calculator(double defaultResult) {
+
+        this.result = defaultResult;
+    }
+
+    void plus(double value) {
+
+        this.result = result + value;
+    }
+
+    void minus(double value) {
+
+        this.result = result - value;
+    }
+
+    void multiply(double value) {
+
+        this.result = result * value;
+    }
+
+    void divide(double value) {
+
+        if (value == 0) {
+            System.out.println("You can't divide by zero!");
+            System.exit(0);
+        }
+        this.result = result / value;
+    }
+
+    double getResult() {
+
+        return this.result;
+    }
+
+    void setResult(double input) {
+
+        this.result = input;
+    }
+
+    void getStoredResults() throws IOException {
+
+        fileService.loadResultsFromFile();
+    }
+
+    void storeResults() {
+
+        fileService.storeResultsIntoFile(list);
+    }
+
+    void setStoreResult() {
```

Comment:
název evokuje, že to je setter, což není ... podle zadání by se to správně mělo jmenovsat storeResult a pak to bude dávat i smysl

Reply:
dojde asi ke kolizi s metodou nad tim, nevadi ... klidne tu metodu pojmenuj stejne jako v fileService

Reply:
Opraveno